index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ramiro Aquistapace</title>
+    <link rel="stylesheet" href="style.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+    </style>
+</head>
+<body>
+    <div class="wrapper">
+      <header class="header">
+        <h3 class="mylogo">My Logo</h3>
+        <img alt="" src="Union.png" class="icon" />
+      </header>
+      <div class="container">
```

Comment:
- [ ] You are missing to add this background images, please do so:
![image](https://user-images.githubusercontent.com/38522578/129119121-d8414be5-c89f-4a8e-b79b-23cba8c1fd77.png)